specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/queue.json

```diff
@@ -0,0 +1,570 @@
+          "QueueOperationList": {
+            "$ref": "./examples/QueueOperationList.json"
+          }
+        },
+        "description": "Gets a list of all the queues under the specified storage account",
+        "parameters": [
+          {
+            "$ref": "./storage.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "./storage.json#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$maxpagesize",
+            "in": "query",
+            "type": "string",
+            "description": "Optional, a maximum number of queues that should be included in a list queue response"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "type": "string",
+            "description": "Optional, When specified, only the queues with a name starting with the given filter will be listed."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- returned a list of queues successfully.",
+            "schema": {
+              "$ref": "#/definitions/ListQueueResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ListQueueServices": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/QueueServiceProperties"
+          },
+          "description": "List of queue services returned.",
+          "readOnly": true
+        }
+      }
+    },
+    "QueueServiceProperties": {
+      "properties": {
+        "properties": {
+          "properties": {
+            "cors": {
+              "$ref": "./common.json#/definitions/CorsRules",
+              "description": "Specifies CORS rules for the Queue service. You can include up to five CorsRule elements in the request. If no CorsRule elements are included in the request body, all CORS rules will be deleted, and CORS will be disabled for the Queue service."
+            }
+          },
+          "x-ms-client-flatten": true,
+          "x-ms-client-name": "QueueServiceProperties",
+          "description": "The properties of a storage account’s Queue service."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Resource"
+        }
+      ],
+      "description": "The properties of a storage account’s Queue service."
+    },
+    "Queue": {
```

Comment:
.net SDK generation/build failed with :
> https://dev.azure.com/azure-sdk/internal/_build/results?buildId=374038&view=logs&jobId=b6885ff5-5b63-57e7-07bc-9d8d90d734f1&j=b6885ff5-5b63-57e7-07bc-9d8d90d734f1&t=edefe71f-a8fe-52c2-c45d-59609a33c1f2

The name "Queue" should be change to some other name.

Reply:
I have changed the name